{{ cookiecutter.repo_name }}/build.gradle.kts

```diff
@@ -2,9 +2,9 @@ import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
 
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 buildscript {
-    repositories {
+     repositories {
```

Comment:
I think we don't need this change, This change does not have proper indentation.